### ISSUE TITLE
Pastel strict dependency prevents upgrading tty-markdown

### ISSUE THREAD

**Issue by DannyBen** [piotrmurach/tty-command#59]
### Describe the problem

After the release of new tty-markdown (with the [kramdown vulnerability fix](https://github.com/piotrmurach/tty-markdown/issues/24)), I am trying to update the bundle of my Jobly gem, which includes both tty-markdown and tty-command, but upgrade is blocked by bundler.

This line:

https://github.com/piotrmurach/tty-command/blob/da308658f0cb96db78d5b3b25985fae0434973a0/tty-command.gemspec#L30

which seems to have been updated [only recently](https://github.com/piotrmurach/tty-command/commit/e7fb8c327ce891f839eafe555e68b08ae2149010#diff-41a3cf09a04512e1420ae686c36c55da), is not yet available in the released version.

So, when trying to update a bundle that includes both tty gems, there is a bundler conflict error.

### Steps to reproduce the problem

Use this Gemfile, and run `bundle`

```
source "https://rubygems.org"
gem "tty-command", '~> 0.9'
gem "tty-markdown", '~> 0.7'
```

### Actual behaviour

Error:

```
Bundler could not find compatible versions for gem "pastel":
  In Gemfile:
    tty-command (~> 0.9) was resolved to 0.9.0, which depends on
      pastel (~> 0.7.0)

    tty-markdown (~> 0.7) was resolved to 0.7.0, which depends on
      pastel (~> 0.8)
```

### Expected behaviour

It is expected to work.

### Describe your environment

* OS version: Ubuntu 18
* Ruby version: 2.7.0
* TTY::Command version:  NA

**Comment by jrochkind** [comment]
One thing this shows is the headaches of using pre-1.0 gems without backwards compat commitments in production. :(

**Comment by goalaleo** [comment]
I have the same problem when trying to update `tty-promt` to `0.22.0`
```
Bundler could not find compatible versions for gem "pastel":
  In snapshot (Gemfile.lock):
    pastel (= 0.7.4)

  In Gemfile:
    tty-command was resolved to 0.9.0, which depends on
      pastel (~> 0.7.0)

    tty-prompt (>= 0.22) was resolved to 0.22.0, which depends on
      pastel (~> 0.8)

Running `bundle update` will rebuild your snapshot from scratch, using only
the gems in your Gemfile, which may resolve the conflict.
```

**Comment by piotrmurach** [comment]
@jrochkind One thing your comment shows is lack of consideration and empathy. These are challenging times and the last thing to deal with is reading unhelpful and uncivil comments like this. I receive plenty of emails, issue reports and pull requests on all my open source projects to review. It is a lot of pressure to handle them all and hence I prioritise things best I can and would appreciate more thoughtful comments. All of my `tty` gems are below `1.0` but I consider them all production-worthy. I put a lot of effort into maintaining backwards compatibility and it's not my intention for my gems to be 'giving headaches' when upgrading.

@DannyBen Thank you for, as always, great report. I'm aware of this problem and it's on my list. As a way of explanation, I'm almost done upgrading all tty gems(26) to have fewer dependencies and be easier to upgrade. This meant changing `pastel` version to be less restrictive going forward. The last 4 remaining tty gems are (`tty-command`/`tty-progressbar`/`tty-editor`/`tty-table`) and only two of these depend on `pastel` and they are my priority now. Expect a release very soon.

@goalaleo Thanks. Will cut a release soon.

**Comment by jrochkind** [comment]
Sorry @piotrmurach , I don't want to have that effect on you, I failed. I appreciate these gems.

**Comment by piotrmurach** [comment]
Released `v0.10.0`. I do apologise for this late release, it was well overdue. I had to deal with many issues that diverted my attention. The good news is that there is only the `tty-progressbar` that awaits an update.

**Comment by DannyBen** [comment]
Thank you Piotr, we appreciate your contribution to the Ruby ecosystem with all these wonderful gems.
